src/api_codes.js

```diff
@@ -33,6 +33,12 @@ export const rawPreviewURL = (IP) => `http://${IP}:8092/rawstream`;
  */
 export const utcURL = (IP) => `http://${IP}:8092/date?date=`;
 
+/**
+ * @param {string} IP
+ * @returns {string}
+ */
+export const timeZoneURL = (IP) => `http://${IP}:8092/timezone?timezone=`;
+
```

Comment:
Which version of the Dwarf API docs are you using?  I did not see setting timezone mentioned in the 1.0 API docs. https://hj433clxpv.feishu.cn/docx/MiRidJmKOobM2SxZRVGcPCVknQg

Reply:
this function is not documented, (see the request on the fly by using wireshark.